src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
```

Comment:
Could we just return `window.ng` at this point? That also means someone programmatically generating the driver could just call:

```javascript
MyComponent.driver('angular2', ng);
```

Rather than needing to pass in all of the below child dependencies.